health/health.d/dockerd.conf

```diff
@@ -7,5 +7,8 @@ component: Docker
     every: 10s
    lookup: average -10s
      crit: $this > 0
-     info: average number of unhealthy docker containers over the last 10 seconds
+     info: Average number of unhealthy docker containers over the last 10 seconds. \
+           There are unhealthy docker containers. \
+           Some containers are not running due to failed health checks. \
+           To find out unhealthy containers use [docker ps -a].
```

Comment:
This should be either ‘To find unhealthy containers’ or ‘To find out which containers are unhealthy’.